src/test/java/posters/pageobjects/pages/user/AccountOverviewPage.java

```diff
@@ -20,9 +20,10 @@
 
     @Override
     @Step("ensure this is an account overview page")
-    public void isExpectedPage()
+    public AccountOverviewPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.